mrp_production_grouped_by_product/models/mrp_production.py

```diff
@@ -65,9 +65,8 @@ def _get_grouping_target_domain(self, vals):
         if date.hour < pt.mo_grouping_max_hour:
             date_end = date.replace(hour=pt.mo_grouping_max_hour, minute=0, second=0)
         else:
-            date_end = date + relativedelta(days=1)
```

Comment:
Why removing this? Doing a "day+=1" could raise an error (e.g. day 32 won't exist) :)
However instead of `relativedela` we could use Odoo fields API:
```python
date_end = fields.Datetime.add(date, days=1)
```